<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PiecePropertyFormClass</class>
 <widget class="GUIFormBase" name="PiecePropertyFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Piece Property</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QToolButton" name="toolButtonArrange">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Piece Arrange</string>
    </property>
    <property name="icon">
     <iconset resource="PieceProperty.qrc">
      <normaloff>:/Qtres/Arrange.png</normaloff>:/Qtres/Arrange.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonCutPiece">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cut Piece &amp; Edit</string>
    </property>
    <property name="icon">
     <iconset resource="PieceProperty.qrc">
      <normaloff>:/Qtres/Cut.png</normaloff>:/Qtres/Cut.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameAlgorithmBase">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>46</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QToolButton" name="toolButtonArrange_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Piece Arrange</string>
    </property>
    <property name="icon">
     <iconset resource="PieceProperty.qrc">
      <normaloff>:/Qtres/Arrange.png</normaloff>:/Qtres/Arrange.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>401</width>
     <height>731</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Manual arrange</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>398</width>
       <height>341</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QFrame" name="frameCategory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>381</width>
        <height>141</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frameGrid">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>191</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <widget class="QTableView" name="tableViewMList">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>191</width>
         <height>181</height>
        </rect>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="labelTopview">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>191</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButtonExecuteAlignment">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Execute alignment in selected items</string>
     </property>
     <property name="icon">
      <iconset resource="PieceProperty.qrc">
       <normaloff>:/Qtres/Select.png</normaloff>:/Qtres/Select.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonUpdate">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>480</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
     <property name="icon">
      <iconset resource="PieceProperty.qrc">
       <normaloff>:/Qtres/Update.png</normaloff>:/Qtres/Update.png</iconset>
     </property>
    </widget>
    <widget class="QTableWidget" name="tablePieceArrange">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>510</y>
       <width>391</width>
       <height>191</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Page</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PieceID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PieceLib</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Angle</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Pos</string>
      </property>
     </column>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>350</y>
       <width>391</width>
       <height>121</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QSpinBox" name="spinBoxAutoCountX">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxAutoCountY">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButtonStartMatrixArrange">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Start arrange</string>
      </property>
      <property name="icon">
       <iconset resource="PieceProperty.qrc">
        <normaloff>:/Qtres/Start.png</normaloff>:/Qtres/Start.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Horizontal (X) count</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Vertial (Y) count</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>381</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Matrix arrangement</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Select piece master before start</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Lib-&gt;Automatic arrange</string>
    </attribute>
    <widget class="QFrame" name="frameGridAuto">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frameCategoryAuto">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelTopviewAuto">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetAutoCurrent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>191</width>
       <height>111</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MasterName</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButtonAutoPopLoad">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>↓</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPushBask">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>↑</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>今回探索配置部品</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAutoExecute">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>配置実行</string>
     </property>
     <property name="icon">
      <iconset resource="PieceProperty.qrc">
       <normaloff>:/Qtres/Start.png</normaloff>:/Qtres/Start.png</iconset>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetAutoResult">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>401</y>
       <width>181</width>
       <height>171</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Page</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Layer</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Count</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>380</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Arranemental result</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Automatic search</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>１:探索元領域を指定</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>２：探索開始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonStartAutoSearch">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Start search</string>
     </property>
     <property name="icon">
      <iconset resource="PieceProperty.qrc">
       <normaloff>:/Qtres/Search.png</normaloff>:/Qtres/Search.png</iconset>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetAutoSearch">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>341</width>
       <height>231</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Page</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Location</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Matching</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>３：探索結果</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>50</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>４：処理</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCopyByAutoSearch">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>410</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy item</string>
     </property>
     <property name="icon">
      <iconset resource="PieceProperty.qrc">
       <normaloff>:/Qtres/Copy.png</normaloff>:/Qtres/Copy.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PieceProperty.qrc"/>
 </resources>
 <connections/>
</ui>
